### DISCUSSION TITLE
energy_loss_percent selbst errechnen

### DISCUSSION THREAD

**Discussion by Karkas66** [butterwecksolutions/Spotmarket-Switcher#166] [Q&A]
Moin,

in der Konfig gibt es ja den Punkt 
`energy_loss_percent=23.3                      # Enter how much percent of the energy is lost by the charging and discharging process.`
Du schreibst aber nicht dazu wie man den selbst errechnen kann. Der ist ja sicherlich abhängig vom Multiplus Modell.

Hast du da einen Anhaltspunkt?

**Comment by butterwecksolutions** [comment]
Hi, das sind Messwerte, die andere Nutzer mal ermittelt bzw. über ein Jahr gemessen hatten.
Ich habe mal die Quelle rausgesucht wo ich den Wert damals gefunden hab und halte ihn für realistisch im AC-AC Betrieb.
https://meintechblog.de/2022/02/08/operation-hausspeicher-wirkungsgrad-meines-ac-seitig-eingebundenen-ess/#comment-14101

Gruß!
Christian
